build/Targets/TransformVsixManifest.xslt

```diff
@@ -0,0 +1,14 @@
+<xsl:stylesheet version="1.0"
```

Comment:
You might want to add a comment for literally everyone else that says what this does in actual English.